### PR TITLE
Writing Day: Add Good Docs and Mutual Aid session

### PR DESCRIPTION
**Pull request by rosewms**
Added the first and second sessions explanation and added times for clarity. Kenzie's project is second session only.

----
:books: Documentation preview :books:: https://writethedocs-www--2007.org.readthedocs.build/

### CODE REVIEW

**Review by barbaricyawps — APPROVED** [review]
@rosewms , this looks great to me. Thanks!

File: docs/conf/atlantic/2023/writing-day.rst [hunk 1]
```diff
@@ -12,15 +12,58 @@ Schedule
 - Date & Time: **{{date.day_two.dotw}}, {{date.day_two.date}}, {{date.day_two.writing_day_time}} {{tz}}**.
 - Location: **{{about.venue}}**.
 
-Project list
-------------
+Writing Day projects are welcome to join us for the entire event, the first session, or the second session.
```
Comment (plaindocs) [inline docs/conf/atlantic/2023/writing-day.rst:15]: Or both?

**Review by mxsasha — APPROVED** [review]
💯

**Review by plaindocs — APPROVED** [review]

File: docs/conf/atlantic/2023/writing-day.rst (hunk 1 above)
Comment (mxsasha) [reply docs/conf/atlantic/2023/writing-day.rst:15]: It says that by "for the entire event", right?

Comment (plaindocs) [reply docs/conf/atlantic/2023/writing-day.rst:15]: lol, sure does :-D

**Pull request merged**